Android/app/src/main/java/com/example/kahye/common/Adapter.java

```diff
@@ -23,29 +24,33 @@
 
 public class Adapter extends PagerAdapter {
     private ClassList classList;
-    private String selectedDate;
     private Context context;
+    private String selectedDate;
     private LayoutInflater inflater;
     private Class selectedClass;
-    //Todo(woongjin) change the hardcoded url to read config file and use it
-    private String baseImgUrl= "http://10.0.2.2:8000";
+    //Todo(woongjin) change the hardcoded url to read config file and
+    // use it
+    private String baseImgUrl= "http://52.8.187.167:8000";
 
     ImageButton classButton;
+    TextView classTextView;
+    TextView expertTextView;
 
-    //TODO(woongjin) get images from server
-    private Integer [] images = {R.drawable.coffee, R.drawable.cooking, R.drawable.wine};
     private String [] imagesURL = {};
     private String [] classes = {};
+    private String[] expertNameList = {};
 
-    public Adapter(Context context, ClassList classList, String[] imagesURL, String selectedDate) {
+    public Adapter(Context context, ClassList classList, String[] imagesURL,
+                   String[] expertNameList, String selectedDate) {
         this.context = context;
         this.classList = classList;
         this.imagesURL = imagesURL;
+        this.expertNameList = expertNameList;
```

Comment:
@Kahyelee I just started to take a look at this. Why do we have **classList**, **imagesURL** and **expertNameList** here? I thought that class object contains its corresponding image and an instructor name.

Reply:
I changed the code. The classList variable is the list of classes, so it is needed to show the information about the class in adapter. I deleted the imagesURL and expertNameList variables.

Reply:
I changed the code in my local if the request is merged I will make pull request about this issue #24